VineeTagarwaL-code/30-Java-Projects
# Add new Java project: Snake Game

# Pull Request: Add Snake Game Project in Java

## Introduction
This pull request adds a Java-based Snake Game project to the repository, showcasing programming concepts like functions, GUI, library imports, inheritance, and arrays.

## Purpose
The purpose is to provide an educational resource for new Java developers, highlighting practical usage of these concepts in a game development context.

## Review Instructions
Reviewers, please assess:
1. Code quality and organization.
2. Game functionality, including user interactions.
3. Effective demonstration of key programming concepts.
4. Proper resource management and code structure.

## How to Test
1. Ensure Java and an IDE are installed.
2. Run Welcome.java.
3. Play using arrow keys or WASD keys.
4. Avoid collisions with the snake itself.

Thank you for your review.

Sincerely,
GenDelta.
